lms/djangoapps/lti_provider/users.py

```diff
@@ -15,8 +15,12 @@
 from lti_provider.models import LtiUser
 from student.models import UserProfile
 
+USERNAME_SIZE = 30 #size of the field in the database
```

Comment:
Perhaps name the constants more clearly so that the comment isn't necessary?  Something like `USERNAME_DB_FIELD_SIZE`?